scripts/upload-source-maps.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { execSync } from 'child_process'
+import { readdirSync, unlinkSync } from 'fs'
+import { join } from 'path'
+
+const main = () => {
+	if (process.env.VERCEL_ENV === 'development') {
+		return
+	}
+
+	const LATEST_SHA = process.env.VERCEL_GIT_COMMIT_SHA
+	// const PATH_PREFIX = 'https://developer.hashicorp.com/_next/static/'
+	// const SERVICE = 'developer.hashicorp.com'
+	const PATH_PREFIX =
+		'https://dev-portal-git-rn-featbuild-source-maps-during-deploy-hashicorp.vercel.app/_next/static/'
```

Comment:
I think this'll need to somehow account for variable deployment URLs!

TLDR I think we could do like:

```suggestion
	const PATH_PREFIX = '/_next/static/'
```

In more detail:

- One option is to use the full path, like `https://${process.env.VERCEL_URL}/_next/static/`... though this won't work for testing dry runs in local dev (maybe a non-issue)
- Another option is to use an absolute path ([docs here](https://github.com/DataDog/datadog-ci/tree/master/src/commands/sourcemaps#commands), which feels maybe more convenient?

Reply:
I ended up going with process.env.VERCEL_ENV === 'production' ? 'https://developer.hashicorp.com/_next/static/' : `https://${process.env.VERCEL_BRANCH_URL}/_next/static/` here